ApiTests/Tests/API/APITests.cs

```diff
@@ -1,44 +1,58 @@
+using RestSharp;
+using System.Net;
 using TestProject.Models;
 using TestProject.Utils;
 
+
 namespace TestProject.Tests.API
 {
     public class Tests
     {
         [SetUp]
         public void Setup()
-        {
-            //move test data creation here
-        }
-
-        [Test]
-        public void PetTest()
         {
             //create a pet using post request
+            //move test data creation here
```

Comment:
once the part is done it's better to remove comment. 
Usually the approach is that we only comment methods that are difficult to understand (too much logic or similar to that)